.github/workflows/unit-tests.yml

```diff
@@ -3,13 +3,11 @@ on: [push, pull_request]
 jobs:
   container-test-job:
     runs-on: ubuntu-latest
-    container: maierbn/opendihu:latest
+    container: carmehp/opendihu:ci
```

Comment:
That's an unusual tag that I would not know how to interpret without reading some documentation. Using tags to describe two very different images would probably be confusing. Why not just call the one `carmehp/opendihu-ci:latest`?